week5.ipynb

```diff
@@ -0,0 +1,366 @@
+{
```

Comment:
<p>This intro section looks good! Only thing is that I notice some words capitalized that I don't think need to be; if the intention is to provide emphasis on those terms as vocab words, maybe use italics or boldface instead?</p><p> </p><p>Also, in the paragraph that starts "In this form, the S-B equation compares...", you're missing a closing parenthesis in the second sentence.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CUB-Computational-Tools/2020-ps-jolsenvaldez/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>